cfme/tests/cloud/test_instance_power_control.py

```diff
@@ -94,13 +95,12 @@ def _fail_func():
 
 
 def wait_for_pwr_state_change(instance, state_change_time, timeout=720):
```

Comment:
Thoughts on this being a method of `BaseEntity`?

Reply:
Also do you think the distinction between this method and `wait_for_power_state_change_rest` could be made a bit clearer by calling this method `wait_for_power_state_refresh`?

Reply:
> Thoughts on this being a method of BaseEntity?

Probably not, because we already have methods to check power state change viaUI and viaREST, we rarely have the need to check `last_changed_on` property.